src/pseudopeople/dataset.py

```diff
@@ -145,7 +145,7 @@ def _reformat_dates_for_noising(self) -> None:
                     # https://github.com/pandas-dev/pandas/issues/44764
                     # Year is already guaranteed to be 4-digit: https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#timeseries-timestamp-limits
                     is_na = data[column].isna()
-                    data_column = data.loc[~is_na, column]
+                    data_column = pd.to_datetime(data.loc[~is_na, column])
```

Comment:
Clarify that we are already doing what's mentioned in the comment

Reply:
This was working with the sample data so I've removed it for now and if it comes up again I'll take note of which data are being problematic